tools/accuracy_checker/accuracy_checker/data_readers/README.md

```diff
@@ -1,14 +1,19 @@
 # Data Readers
 
-Data Reader is a function for reading input data.
-You can use 2 ways to set data_reader for dataset:
+Data Reader is a function for reading input data in a specific format and converting it to a format suitable for model inference. Readers may have parameters available for configuration. The reader and its parameters, if necessary, are set through the configuration file.
```

Comment:
I think data conversion is extra info, which cam mislead users
```suggestion
Data Reader is a class for reading input data in a specific format. Readers may have parameters available for configuration. The reader and its parameters, if necessary, are set through the configuration file.
```

Reply:
Done